pystarport/cluster.py

```diff
@@ -823,6 +823,7 @@ def create_account(cli, account, use_ledger=False):
                 "commission_rate",
                 "details",
                 "security_contact",
+                "gas_prices"
```

Comment:
Seems no gas related field in [MsgCreateValidator](https://github.com/cosmos/cosmos-sdk/blob/release/v0.50.x/x/staking/types/tx.pb.go#L42), may i ask why need change gas-prices for gentx?

Reply:
yes, is related to the discussion here https://github.com/evmos/evmos/pull/1864
When setting a base fee > 0 on genesis, the gentx should have the corresponding gas price. Otherwise will fail with insufficient fee error on chain initialization.

Reply:
can you help to sign the commits?